js/script.js

```diff
@@ -43,18 +37,23 @@ window.addEventListener('DOMContentLoaded', () => {
         return res.json(); // JSON в обычный формат JS перевожу
     }
 
+    // console.log(getResource('https://swapi.dev/api/people/'))
     // пришлось колхозить с глобальными переменными так как используются в нескольких функциях
     let cardsContainer;
     let cardList;
     let slideWidth;
     let slideIndex = 0;
-
+//Передаю данные в класс
     getResource('https://swapi.dev/api/people/')
         .then(data => {
-            data.results.forEach(({name, height, mass, hair_color, skin_color, eye_color, birth_year, gender}) => {
-                new MenuCard(name, height, mass, hair_color, skin_color, eye_color, birth_year, gender, '.cards').render();
+            data.results.forEach((card) => {
```

Comment:
Вот тут бы я сделал немного подругому

```suggestion
            this.cards = data.results.map((card) => {
                const instance = new createPage(card, '.cards');
                
                instance.render();
                
                return instance;
            });
```

Тут ты как бы сохраняешь список инстансов карточек на уровне страницы, чтобы потом с ними можно было взаимодействовать и вызвать какие-то методы у них